### PR TITLE
Add controllers to fetch skills and hobbies

### PR DESCRIPTION
**Pull request by MFJess**
Why:

* We need to display user's skills, hobbies and company.

This change addresses the need by:

* Creating a controller for each one of those to render this information given a specific user.

### CODE REVIEW

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    render json: @companies
+  end
+
+  # GET /companies/1
+  def show
+    render json: @company
+  end
+
+  # POST /companies
+  def create
```
Comment (zamith) [inline app/controllers/companies_controller.rb:17]: Do we need to expose this?

File: app/controllers/companies_controller.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  def update
+    if @company.update(company_params)
+      render json: @company
+    else
+      render json: @company.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /companies/1
+  def destroy
```
Comment (zamith) [inline app/controllers/companies_controller.rb:37]: Do we need to expose this?

File: app/controllers/companies_controller.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    if @company.save
+      render json: @company, status: :created, location: @company
+    else
+      render json: @company.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /companies/1
+  def update
```
Comment (zamith) [inline app/controllers/companies_controller.rb:28]: Do we need to expose this?

**Review by jopedroliveira — APPROVED** [review]

**Comment by jopedroliveira** [comment]
Lgtm. Check the build errors
`app/serializers/user_serializer.rb:10:1: Layout/EmptyLinesAroundClassBody: Extra empty line detected at class body end.`

**Comment by MFJess** [comment]
Checked ✅

**Pull request merged**